pkg/api/v1/types.go

```diff
@@ -624,10 +624,12 @@ type Container struct {
 	Name string `json:"name" description:"name of the container; must be a DNS_LABEL and unique within the pod; cannot be updated"`
 	// Required.
 	Image string `json:"image" description:"Docker image name"`
-	// Optional: The docker image's entrypoint is used if this is not provided; cannot be updated.
-	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; the docker image's entrypoint is used if this is not provided; cannot be updated"`
-	// Optional: The docker image's cmd is used if this is not provided; cannot be updated.
-	Args []string `json:"args,omitempty" description:"command array; the docker image's cmd is used if this is not provided; arguments to the entrypoint; cannot be updated"`
+	// Optional: The docker image's entrypoint is used if this is not provided; cannot be updated;
+	// variable references ("$(VAR_NAME)") are expanded according to the expansion spec.
+	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; the docker image's entrypoint is used if this is not provided; cannot be updated; variable references are expanded according to the expansion spec"`
```

Comment:
Can we not also include the format example in this json? My (possibly mistaken) understanding is that this is the only part that people looking at the API via swagger actually see